app/metrics.yaml

```diff
@@ -1415,11 +1527,13 @@ custom_tabs_toolbar:
       - https://github.com/mozilla-mobile/focus-android/issues/5649
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/pull/5748
+      - https://github.com/mozilla-mobile/focus-android/pull/7906
     data_sensitivity:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-    expires: 107
+      - rtestard@mozilla.com
+    expires: never
   action_button_tapped:
```

Comment:
The new data_reviews link is missing on this one :)

Reply:
Good catch! thanks, I'll fix.